sequence_task_3/student/tests/unit_tests/src/maintest.cpp

```diff
@@ -25,7 +25,21 @@ TEST(Reverse, TrailingSpaces)
 
 TEST(Reverse, Corner)
 {
-    const std::string input {""};
-    const std::string expected {""};
+    const std::string input { "" };
+    const std::string expected { "" };
+    EXPECT_EQ(reverseWords(input), expected);
+}
+
+TEST(Reverse, Corner_2)
```

Comment:
Thanks, could you please create a new pull request for these tests?

Reply:
Done
https://github.com/smay1613/LTC-STL-Student/pull/148